aptos-move/block-executor/src/executor.rs

```diff
@@ -750,6 +757,9 @@ where
                 ExecutionStatus::Abort(_) => {
                     txn_commit_listener.on_execution_aborted(txn_idx);
                 },
+                ExecutionStatus::DirectWriteSetTransactionNotCapableError => {
+                    panic!("Cannot be materializing with DirectWriteSetTransactionNotCapableError");
```

Comment:
panic? Shouldn't we fall back to sequential execution?

Reply:
this means we are committing transaction that raised this error - this should never happen.

instead executor will catch it in execute(), and fallback to sequential